src/Sylius/Component/Core/Model/AdminUser.php

```diff
@@ -80,4 +83,15 @@ public function setLocaleCode(?string $code): void
     {
         $this->localeCode = $code;
     }
+
+    public function getAvatar(): ?ImageInterface
+    {
+        return $this->avatar;
+    }
+
+    public function setAvatar(?ImageInterface $avatar): void
+    {
+        $avatar->setOwner($this);
```

Comment:
The owner is not persisted in the database so I think we don't need to set it anyway.

Reply:
This steep is needed because when i remove this line, I get `Fatal error: Call to a member function getPath() on null (Behat\Testwork\Call\Exception\FatalThrowableError)` during execute `Then I should see this administrator account with avatar "troll.jpg"` step, 

Similar `setOwner($this)` sytuation occudrred 
https://github.com/Sylius/Sylius/blob/83e1915d99837b5a14adf8145f210bfbb96e06b5/src/Sylius/Component/Core/Model/Taxon.php#L74

Our mapping `AvatarImage` : `<one-to-one field="owner" target-entity="Sylius\Component\Core\Model\AdminUser" inversed-by="avatar">` use `owner` field to 
make relation?

AdminUser : `<one-to-one field="avatar" target-entity="Sylius\Component\Core\Model\AvatarImage" mapped-by="owner" orphan-removal="true">
`



If I'm wrong, correct me ;)